trieste/utils/pareto.py

```diff
@@ -148,3 +148,42 @@ def hypervolume_indicator(self, reference: TensorType) -> TensorType:
         hypervolume = tf.reduce_sum(tf.reduce_prod(upper - lower, 0))
 
         return tf.reduce_prod(reference[None] - min_pfront) - hypervolume
+
+    def get_hypercell_bounds(self, anti_reference: TensorType, reference: TensorType) -> Tuple:
```

Comment:
```suggestion
    def get_hypercell_bounds(self, anti_reference: TensorType, reference: TensorType) -> tuple[TensorType, TensorType]:
```

Reply:
you'll also need `from __future__ import annotations` before your import statements for this to work

Reply:
Is it a good idea to directly make use of `Tuple[TensorType, TensorType]`?

Reply:
probably not really - we've migrated away from using `typing.Tuple` (except for type aliases where that's not possible yet)